packages/dropdown/src/Default/styles.module.scss

```diff
@@ -0,0 +1,110 @@
+@use '@owlui/theme/src' as theme;
+
+.dropdown {
+  $ref: &;
+  padding: 0;
+
+  .dropdown-item:focus {
+    background-color: white;
+    color: black;
+    outline-color: #0083C3;
+  }
+
+  button.active {
+    background-color: #0083C3 !important;
+    border-color: #0083C3;
+    color: white !important;
+  }
+
+  #{$ref}-toggle.btn-primary {
```

Comment:
Let's use the default BS styles, no need to override them at this point. Once Scott provides us with the final designs we can create new variants with proper styles. 

I'd suggest you remove most of these styles and see how it looks by default, and then add only the styles needed to make your component work.